docs/built_with.rst

```diff
@@ -0,0 +1,20 @@
+Built with aiohttp
+==================
+
+aiohttp is used to build useful libraries built on top of it,
+and there's a page dedicated to list them: :ref:`aiohttp-3rd-party`.
```

Comment:
@socketpair @asvetlov it's rendering well on http://aiohttp.readthedocs.io/en/latest/built_with.html

Notice that the "ref" markup is only rendered by Sphinx, not github's restructuredtext renderer